core/shared/platform/zephyr/zephyr_thread.c

```diff
@@ -577,4 +577,35 @@ os_thread_get_stack_boundary()
 
 void
 os_thread_jit_write_protect_np(bool enabled)
-{}
\ No newline at end of file
+{}
+
+int
+os_thread_detach(korp_tid thread)
+{
+    (void)thread;
+    return BHT_OK;
+}
+
+void
+os_thread_exit(void *retval)
+{
+    (void)retval;
+    os_thread_cleanup();
+    k_thread_abort(k_current_get());
+}
+
+int
+os_cond_broadcast(korp_cond *cond)
+{
+    os_thread_wait_node *node;
+    k_mutex_lock(&cond->wait_list_lock, K_FOREVER);
+    node = cond->thread_wait_list;
+    while (node) {
+        os_thread_wait_node *next = node->next;
+        k_sem_give(&node->sem);
+        node = next;
+    }
+    cond->thread_wait_list = NULL;
```

Comment:
Should not set thread_wait_list to NULL? The waiting thread will remove its wait_node from the list, and after each waiting thread removes the node, the thread_wait_list will be NULL, see os_cond_wait_internal.